plugin/web/extensions/sidebar/modals/datacatalog/index.html

```diff
@@ -23,7 +23,11 @@
 window.addEventListener("message", function (e) {
     if (e.source !== parent) return;
     if (e.data.action === "catalogModalOpen" && e.data.payload) {
-        if(e.data.payload.dataset) window.selectedDataset = e.data.payload.dataset;
+        if(e.data.payload.dataset) {
+            window.selectedDataset = e.data.payload.dataset;
+        } else if(e.data.payload.expandedKeys) {
+            window.expandedKeys = e.data.payload.expandedKeys;
```

Comment:
Again, window shouldn't be accessible from the plugin.

Reply:
Done!